src/test/web/messages/alternate-message-resources-annotated.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+normal_message=normal_message
```

Comment:
Just for a point of clarity, if the key and value are the same, and I had code (in theory) to fall back to the key if the value is not found, this would be difficult to debug.

Can we use:
```
normal-message=Normal message
```

or something like that to make it more real world and more obvious if we were to ever assert on the key by accident?

Reply:
Fixed